SEGGER/SEGGER_RTT.c

```diff
@@ -1892,6 +1893,28 @@ int SEGGER_RTT_SetFlagsDownBuffer(unsigned BufferIndex, unsigned Flags) {
   return r;
 }
 
+#if defined(CONFIG_SEGGER_RTT_INIT_MODE_STRONG_CHECK)
+static void SEGGER_RTT_StrongCheckInit(void) {
+  volatile SEGGER_RTT_CB* p;
+  p = (volatile SEGGER_RTT_CB*)((char*)&_SEGGER_RTT + SEGGER_RTT_UNCACHED_OFF);
+  unsigned i;
+  bool DoInit = false;

```

Comment:
I needed to include `<stdbool.h>` at the top of this file to avoid this error
```shell
SEGGER/SEGGER_RTT.c:1906:3: error: unknown type name 'bool'
 1906 |   bool DoInit = false;
```

Reply:
Thanks @cfriedt, for signalling this issue. It probably depends on the toolchain, as I didn't have it.
Yet, looking at Seger code, they seem not to use boolean, so I changed it to int.